tests/FSharp.AWS.DynamoDB.Tests/Utils.fs

```diff
@@ -37,14 +39,19 @@ module Utils =
 
 
     type TableFixture() =
+
         let client = getDynamoDBAccount()
         let tableName = getRandomTableName()
+
         member _.Client = client
         member _.TableName = tableName
 
+        member _.CreateContextAndTableIfNotExists<'TRecord>() =
```

Comment:
If the fixture is working as intended the table shouldn’t exist - is it worth just calling this`CreateContextAndTable`?

Reply:
was trying to minimise diff vs what was there. If we don't rename Scripting.TableContext.Initialize, the logical name would be Initialize....

Maybe `CreateEmpty`?

Reply:
Applied CreateEmpty, let me know if you prefer `Initialize`